mne_bids/utils.py

```diff
@@ -503,6 +503,29 @@ def _get_brainvision_paths(vhdr_path):
     return (eeg_file_path, vmrk_file_path)
 
 
+def copyfile_ctf(src, dest):
+    """Copy and rename CTF files to a new location.
+
+    Parameters
+    ----------
+    src : str
+        path to the source raw .ds folder
+    dest : str
+        path to the destination of the new bids folder.
+    """
+    sh.copytree(src, dest)
+    # list of file types to rename
+    file_types = ('.acq', '.eeg', '.hc', '.hist', '.infods', '.infods.bak',
+                  '.meg4', '.newds', '.res4')
+    # Rename files in dest with the name of the dest directory
+    files_to_rename = [f for f in os.listdir(dest) if f.endswith(file_types)]
+    bids_folder_name = op.splitext(op.split(dest)[-1])[0]
+    for ff in files_to_rename:
```

Comment:
may I suggest better variable names?

`ff` -> `fname`
`files_to_rename` -> `fnames`

what do you think?